ui/css/itcss/components/send.scss

```diff
@@ -515,6 +515,12 @@
     margin-top: 5px;
   }
 
+  &__error-hex-data {
+    @include H8;
```

Comment:
I think this looks a bit small?

![Screen Shot 2022-06-06 at 5 19 49 PM](https://user-images.githubusercontent.com/34557516/172258745-e4a39864-13fa-49a5-82ce-230a2e494036.png)

looks like the `error-amount` is `H7`

Reply:
I made it H8 because H7 makes is 2 lines. 
@georgewrmarshall any comments?

Reply:
Hello! H7 should be correct we want all our error text under inputs to be the same font-size. If it wraps on to 2 lines that's ok.